ably.d.ts

```diff
@@ -4,478 +4,480 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 declare namespace ablyLib {
-  namespace ChannelState {
```

Comment:
I really dislike that you have changed indentation and made changes in code in the same commit.  As a result, I have no way to review what changed here as everything other than `declare namespace ablyLib` has now changed.  Please have separate commits to change whitespace / indentation from other commits.

Reply:
P.S. I can review with `?w=1` to hide whitespace, but then I cannot add comments to commit lines in that same view

Reply:
@mattheworiordan This is all quite experimental. I don't know why I asked for a review. It's definitely not ready for review.